src/esp/bindings/SensorBindings.cpp

```diff
@@ -188,8 +190,102 @@ void initSensorBindings(py::module& m) {
           R"(The distance to the near clipping plane for this CameraSensor uses.)")
       .def_property(
           "far_plane_dist", &CameraSensor::getFar, &CameraSensor::setFar,
-          R"(The distance to the far clipping plane for this CameraSensor uses.)");
-
+          R"(The distance to the far clipping plane for this CameraSensor uses.)")
+      .def(
+          "buffer",
+          [](CameraSensor& self, int gpuDevice) {
+            py::handle handle = py::cast(self);
```

Comment:
Why do you need to cast this back to a python object? It should automatically be cast when called on any relevant python function. Also the naming is a tad nondescript.